### DISCUSSION TITLE
I can't claim my GitHub student benefits for my university id

### DISCUSSION THREAD

**Discussion by dineshWic** [community/community#63251] [GitHub Education]
### Hi everyone,

Previously I had two GitHub accounts one for my personal mail and the other for university mail. I deleted my student GitHub account and added that mail to my personal GitHub account.

Now my personal GitHub account has both emails and I want to get student benefits for that account. But when I select mail for student benefits, It gives this message "2020*****@stu.**********.lk has already been used to obtain an academic discount on GitHub account(s) dinesh200034". "dinesh200034" is the name of deleted GitHub account. How to fix this Issue?

Note: This issue has not affected for my colleges.

**Comment by humairshoukat** [comment]
Try submitting your University ID card picture. Try taking the picture from your phone while submitting the picture.
